apps/site/assets/ts/components/Badge.tsx

```diff
@@ -2,14 +2,17 @@ import React from "react";
 
 const Badge = ({
   text,
-  contextText
+  contextText,
+  bgClass
 }: {
   text: string;
   contextText?: string;
+  bgClass?: string;
 }): JSX.Element => {
   return (
     <div
-      className="u-error-background font-weight-bold ps-8 pe-8 fs-14"
+      className={`${bgClass ||
+        "u-error-background"} font-weight-bold ps-8 pe-8 fs-14`}
```

Comment:
nit-pick(non-blocking): Could help with readbility/wrapping to make this its own variable above.